src/components/policy/policy_external/include/policy/policy_helper.h

```diff
@@ -201,6 +201,15 @@ struct CheckAppPolicy {
    */
   bool IsRequestSubTypeChanged(const AppPoliciesValueType& app_policy) const;
 
+  /**
+   * @brief IsEncryptionRequiredFlagChanged check if encryption_needed flag was
+   * changed for application or application groups
+   * @param app_policy applicaiton policies
```

Comment:
applicaiton -> application

Reply:
fixed in f87bb9a5507d9e710839ecf9bae2fd513bab2f06